CHANGELOG.md

```diff
@@ -20,8 +46,9 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Add country selection screen after document selection. This screen is skipped by default for a preselected document but can still be displayed by enabling the `showCountrySelection` option for the `document` step.
 - UI: New warnings for cut-off & blurry images detection.
 - UI: When the uploaded image is either cut-off, glary or blurry, the end-user must retry at most 2 times prior to proceeding further.
-- Internal: The release script and the `release/RELEASE_GUIDELINE.md` file now include the information needed to update the `MIGRATION.md` file.
 - UI: Added Residence Permit option for document selection
+- Internal: The release script and the `release/RELEASE_GUIDELINE.md` file now include the information needed to update the `MIGRATION.md` file.
+- Internal: Send additional `system` data in `sdk_metadata` which contains `os`, `os_version`, `browser` & `browser_version` info of the current session.
```

Comment:
Was this part of version `6.0.0`?

Reply:
It wasn't part of the changelog when the release was made https://github.com/onfido/onfido-sdk-ui/releases